pkg/wallet/wallet.go

```
@@ -133,5 +166,62 @@ func New(nickname string) (*Wallet, error) {
 			Salt:       salt,
 			Nonce:      nonce,
 		}},
-	}, nil
+	}
+
+	wallets := readWallets()
+	wallets = append(wallets, wallet)
+
+	bytesOutput, err := json.Marshal(wallets)
```

the error is never checked